### DISCUSSION TITLE
Celebration of the 1st ArozOS Stable Release

### DISCUSSION THREAD

**Discussion by tobychui** [tobychui/arozos#52] [Show and tell]
(Chinese version available below 下方附有中文版本)
![celebrate](https://user-images.githubusercontent.com/24617523/107776774-f644ac00-6d7c-11eb-93f9-29b09e28bfd9.png)

## The First ArozOS Stable Release - v1.110
Thanks for all the stars and fork since 2018. We are please to walk through such a long period of time with all you guys. This is our first stable release of the ArozOS 1.0 since 2016 (and since we moved to Github from 2018) . 

### Introducing Our Mascot - Aroz
Aroz is our system Mascot since 2016 and she always comes up whenever you encounter errors or failure in your system. You might have seen her once or twice before. With the first stable release of ArozOS, we, the core developers of ArozOS, hired an artist to update her images from the original sketches. Now, you can see her grown up with us in the illustration above. 

### Coming Next? 

From v1.111 onward, ArozOS system will start to focus on Cluster and IoT development. Making use of Raspberry Pis and the power of ESP Micro-controller, we will continue to provide completely personal and private cloud solution for you, making sure your files, documents, music, videos, smart light / switches, speakers and all your electronics will still work without the tech giants (even without internet!), while keeping your privacy in your own hand. 

Finally, wish everyone a Happy Lunar New Year!!

Toby Chui 
Software Developer from Hong Kong
12 Feb 2021

**Comment by tobychui** [comment]
![celebrat_zhe](https://user-images.githubusercontent.com/24617523/107776956-30ae4900-6d7d-11eb-960a-7b397e9f8084.png)

## 首個 ArozOS 穩定版發佈 - v1.110
感謝各位由 2018 年以來的每個 Star 跟 Fork。我們很高興可以與你們分享這個開源的雲端系統開發計劃。 
是次是我開發的 ArozOS 1.0 版本自 2016 年（或自 2018 年搬到 Github 以後）第一個發佈的穩定版本。

## 向你們介紹一下 ArozOS 系統的吉祥物 - Aroz
Aroz 是自 2016 年我這套系統開發以來就一直存在的吉祥物。你有時候可以在系統的錯誤頁面見到她，所以說不定你已經見過她呢（？）
因為是次發佈是我們首次發佈的穩定版，我們幾位主要開發者決定請一位繪師為她重設計一個長高了一點的形象，而結果就是你們上面看到的這張立繪了。

## 之後的發展？
自 v1.111 及之後的版本，ArozOS 將會主要開發與 集群 及 物聯網 連接相關的部分。使用 Raspberry Pi 跟 ESP 控制器，我們希望可以為使用此系統的各位提供一個完全 私人、個人化的雲端生態圈，讓你可以不受限於科技公司（甚至無互聯網連接！）之下存取你的檔案、文件、音樂、影片、控制你的智能家居系統、智能喇叭等等，讓你全權掌握屬於你的個人資料。

最後，恭祝大家農曆新年快樂！！

Toby Chui
來自香港的開源軟體開發者
2021 年 2月 12日

**Comment by CrazyCoder009** [comment]
Really love your hardwork man! Keep it up! I was personally also working on something similar.

**Comment by nequals** [comment]
You really did a great job. There is no other functioning system with a stable release out there. Keep up the good work man!
